code/ui/components/tabs/Tab.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+using Sandbox.UI;
+using Sandbox.UI.Construct;
+
+namespace TTTReborn.UI
+{
+    public partial class Tab : TTTPanel
+    {
+        public readonly Label TitleLabel;
+        public readonly Tabs Tabs;
```

Comment:
Should these params be `public`? Would there be cases where we need this data available?

Reply:
I'm wondering if there should be a better name for `Tabs` here. Maybe something to indicate that `Tabs` has ownership of `Tab`?